ci-operator/jobs/openshift/kubernetes/openshift-kubernetes-master-presubmits.yaml

```diff
@@ -924,23 +924,23 @@ presubmits:
       - name: result-aggregator
         secret:
           secretName: result-aggregator
-    trigger: (?m)^/test( | .* )e2e-aws-ovn-downgrade,?($|\s.*)
+    trigger: (?m)^/test( | .* )e2e-aws-ovn-fips,?($|\s.*)
   - agent: kubernetes
     always_run: true
     branches:
     - ^master$
     - ^master-
     cluster: build05
-    context: ci/prow/e2e-aws-ovn-fips
+    context: ci/prow/e2e-aws-ovn-runc
     decorate: true
     labels:
       ci-operator.openshift.io/cloud: aws
       ci-operator.openshift.io/cloud-cluster-profile: aws
       ci.openshift.io/generator: prowgen
       pj-rehearse.openshift.io/can-be-rehearsed: "true"
-    name: pull-ci-openshift-kubernetes-master-e2e-aws-ovn-fips
```

Comment:
did you mean to replace the fips in the name?

Reply:
No, the `make jobs` changed the order. `crun-->runc`, so it's an alphabetical change.